### ISSUE TITLE
can't run ps command

### ISSUE THREAD

**Issue by hhuchzh** [MFDGaming/ubuntu-in-termux#26]
when login into the ubuntu system in privilege, run ps to check the process, it gives 
Error: /proc must be mounted.

**Comment by MFDGaming** [comment]
Do you have android 8 or higher?

**Comment by hhuchzh** [comment]
> Do you have android 8 or higher?

yes, I deploy this project in Samsng Glaxy S9，it is android9.

**Comment by hhuchzh** [comment]
> Do you have android 8 or higher?

Yes, I deployed this project in Samsung Galaxy S9, it is android S9

**Comment by MFDGaming** [comment]
That means that it will never work
